tests/daemon/test-persistent-cache.c

```diff
@@ -1121,11 +1063,6 @@ test_persistent_cache_get_offset_can_build_boot_metadata_file (gboolean     *unu
    */
```

Comment:
Not sure if you didn't see this comment last time or chose to leave it for a separate issue:

The comment above that says "Don't write a default boot offset file [...]" should be replaced with:
```g_assert_cmpint (g_unlink (TEST_DIRECTORY BOOT_OFFSET_METADATA_FILE), ==, 0);``` (pre-existing defect).